app/models/feedback_submission.rb

```diff
@@ -1,18 +1,32 @@
-class FeedbackSubmission < ActiveRecord::Base
-  validates :body, presence: true
-  validate :validate_email
+class FeedbackSubmission
+  include NonPersistedModel
 
-  before_validation :strip_email_address, on: :create
+  attribute :body, String
+  attribute :email_address, String
```

Comment:
I'm struggling to see the value in validating email addresses against sendgrid for the booking process, but I'm entirely certain that it's unnecessary for the feedback email address.

I do think we should check that the email address is in a valid format, which seems entirely ok to duplicate in public.